docs/decisions/0004-external-event-bus-and-django-signal-events.rst

```diff
@@ -15,19 +15,20 @@ The purpose of this ADR is to better define the relationship between the existin
 
 Decision
 --------
+Note that for purposes of this ADR, ``event`` refers to a dictionary of data that is emitted by a Django signal, specifically by an instance of ``OpenEdxPublicSignal``
 
 - Event definitions will be shared between internal and external events. The event definitions will continue to be in the form of ``OpenEdxPublicSignal``, as decided in ADR ":doc:`0003-events-payload`".
 
-- At this time, event bus events will be triggered from a Django signal handler of the Django signal representing the corresponding internal event.
+- At this time, events will be published to the event bus from a Django signal handler listening to the Django signal sending the original event.
```

Comment:
I think this sentence is still a little unclear. Maybe `At this time, events will be published to the event bus from a Django signal handler listening for particular Django signals.`?